app/models/export/tasks/row.rb

```diff
@@ -0,0 +1,36 @@
+module Export
+  class Tasks
+    class Row
+      # Always 1 for now. May be used to indicate different task types in future.
+      FIXED_TASK_TYPE = 1
+      EMPTY_FOR_NOW = nil
+
+      attr_reader :task
+
+      def initialize(task)
+        @task = task
+      end
+
+      def year_and_month
+        "#{task.period_year}#{format('%02d', task.period_month)}"
+      end
+
+      def row_values
+        [
+          task.id,
+          year_and_month,
+          task.supplier.id,
```

Comment:
could save some database queries by accessing these two ids directly, i.e. `supplier_id` and `framework_id`.

Reply:
Oh excellent spot, thanks.